contracts/token/psp22/traits.rs

```diff
@@ -48,6 +49,14 @@ pub struct PSP22MetadataData {
 
 declare_storage_trait!(PSP22MetadataStorage, PSP22MetadataData);
 
+#[derive(Default, Debug, SpreadLayout)]
+#[cfg_attr(feature = "std", derive(StorageLayout))]
+pub struct PSP22WrappedData {
+    pub underlying: Lazy<AccountId>,
+}
+
+declare_storage_trait!(PSP22WrappedStorage, PSP22WrappedData);
```

Comment:
You didn't create a derive macro for that trait=)

Reply:
In psp22/derive/lib.rs?